internal/engine/upper.go

```diff
@@ -612,17 +617,17 @@ func eventContainsConfigFiles(manifest model.Manifest, e manifestFilesChangedAct
 	return false
 }
 
-func getNewManifestFromTiltfile(ctx context.Context, name model.ManifestName) (model.Manifest, error) {
+func getNewManifestFromTiltfile(name model.ManifestName) (model.Manifest, *newManifestErr) {
```

Comment:
Which would mean changing this to an `error` too.

Reply:
i sorta like having `newManifestError` in the signature so ppl don't ignore it and add a clause later that returns some other `error` -- but also i don't have strong feelings.